rust/hyperlane-core/src/chain.rs

```diff
@@ -83,6 +83,18 @@ pub enum KnownHyperlaneDomain {
     Injective = 6909546,
     InEvm = 2525,
 
+    Ancient8 = 888888888,
+
+    Blast = 81457,
+
+    Mode = 34443,
+
+    Redstone = 690,
+
+    Viction = 88,
+
+    Zetachain = 7000,
+
```

Comment:
nit: can you create a ticket and tag me to reorder these + the below ones alphabetically (will make it easier for us to make sense of it as we keep adding chains)

Reply:
created here https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3961